kyaml/yaml/alias.go

```diff
@@ -33,7 +43,10 @@ var Unmarshal = yaml.Unmarshal
 var NewDecoder = yaml.NewDecoder
 var NewEncoder = func(w io.Writer) *yaml.Encoder {
 	e := yaml.NewEncoder(w)
-	e.SetIndent(2)
+	e.SetIndent(indent)
+	if sequenceIndentationStyle == CompactSequenceStyle {
```

Comment:
What is the reason behind not making this configurable ? Will it be exposed in the future ?

Reply:
kpt doesn't hold strong opinion about seq indentation, but we want minimum disruption to the users. The plan is to identify the existing indentation of package resources and retain it, for which, we need this to be configurable. Let me know your thoughts.

As of now, kpt can't upgrade to latest kyaml v0.11.0

Reply:
I filed https://github.com/kubernetes-sigs/kustomize/issues/4029. @KnVerey PTAL when you can